pkg/controllers/allocation/controller.go

```diff
@@ -105,32 +104,38 @@ func (c *Controller) Reconcile(ctx context.Context, req reconcile.Request) (reco
 		return reconcile.Result{}, fmt.Errorf("building constraint groups, %w", err)
 	}
 
+	// 5. Get Instance Types Options
+	instanceTypes, err := c.CloudProvider.GetInstanceTypes(ctx)
+	if err != nil {
+		return reconcile.Result{}, fmt.Errorf("getting instance types, %w", err)
+	}
+
 	// 5. Binpack each group
 	packings := []*cloudprovider.Packing{}
 	for _, constraintGroup := range constraintGroups {
-		instanceTypes, err := c.CloudProvider.GetInstanceTypes(ctx)
-		if err != nil {
-			return reconcile.Result{}, fmt.Errorf("getting instance types, %w", err)
-		}
 		packings = append(packings, c.Packer.Pack(ctx, constraintGroup, instanceTypes)...)
 	}
 
-	// 6. Create packedNodes for packings and also copy all Status changes made by the
-	//    cloud provider to the original provisioner instance.
-	packedNodes, err := c.CloudProvider.Create(ctx, provisioner, packings)
+	// 6. Create capacity
+	errs := make([]error, len(packings))
+	workqueue.ParallelizeUntil(ctx, len(packings), len(packings), func(index int) {
```

Comment:
maybe we should limit the workers so that we don't get throttled immediately, 5 might be a good starting point based on https://docs.aws.amazon.com/AWSEC2/latest/APIReference/throttling.html#throttling-limits

Reply:
I think this limiting should happen on the cloud provider side. I don't want to artificially limit things on the karp/core side now that we have the async interface. WDYT?

Reply:
yeah definitely on CP side 👍